e2e-polybft/e2e/broadcast_test.go

```diff
@@ -0,0 +1,88 @@
+package e2e
+
+import (
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/ethgo/wallet"
+
+	"github.com/0xPolygon/polygon-edge/e2e-polybft/framework"
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+// TestE2E_Broadcast sends two transactions (legacy and dynamic fees) to the cluster with the 1 amount of eth
+// and checks that all cluster nodes have the recipient balance updated.
```

Comment:
Why don't you send like 2-3 legacy and 2-3 dynamic fees txs to like 2 or 3 different recipients? Just to be sure that everything works for multiple recipients and nonce's and everything else works correctly even for multiple txs

Reply:
Done